Patient Registry System/PatientRegistrySystem.API/Controllers/AccountController.cs

```diff
@@ -113,7 +113,17 @@ public async Task<IActionResult> Register([FromBody] RegistrationViewModel model
             }
             else
             {
-                return BadRequest("Input problem");
+                string messages = "";
+                foreach (var modelState in ViewData.ModelState.Values)
```

Comment:
you can move this logic to a helper, as it can be used in many places

Reply:
and remove the logic from all the other places to use ur helper/extension instead